kontrakter-felles/main/kotlin/no/nav/tilleggsstonader/kontrakter/arena/vedtak/Vedtak.kt

```diff
@@ -40,8 +40,8 @@ enum class Rettighet(
 ) : KodeArena {
     BOUTGIFTER_ARBEIDSSØKERE("TSRBOUTG", "Boutgifter arbeidssøkere", Stønadstype.BOUTGIFTER),
     BOUTGIFTER("TSOBOUTG", "Boutgifter tilleggsstønad", Stønadstype.BOUTGIFTER),
-    DAGLIG_REISE_ARBEIDSSØKERE("TSRDAGREIS", "Daglig reise arbeidssøkere"),
-    DAGLIG_REISE("TSODAGREIS", "Daglig reise tilleggsstønad"),
+    DAGLIG_REISE_ARBEIDSSØKERE("TSRDAGREIS", "Daglig reise arbeidssøkere", Stønadstype.DAGLIG_REISE_TSR),
+    DAGLIG_REISE("TSODAGREIS", "Daglig reise tilleggsstønad", Stønadstype.DAGLIG_REISE_TSR),
```

Comment:
Skrev feil i stad. Den siste må være TSO
```suggestion
    DAGLIG_REISE("TSODAGREIS", "Daglig reise tilleggsstønad", Stønadstype.DAGLIG_REISE_TSO),
```